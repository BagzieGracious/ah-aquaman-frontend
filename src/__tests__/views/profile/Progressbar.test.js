import { shallow } from "enzyme";
import React from "react";
import Progress from "../../../views/profile/Progress";

describe("Home view component", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Progress />);
    expect(wrapper).toMatchSnapshot();
  });
});
