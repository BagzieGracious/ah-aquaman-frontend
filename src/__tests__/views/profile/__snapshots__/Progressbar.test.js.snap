// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProgressBar component should render without crashing 1`] = `ShallowWrapper {}`;
